README.md

```diff
@@ -1,21 +1,94 @@
+ 
+*Bootstrap group assignments in a snap.* Invite students to a shared repository, and cap the number of students per group. Use the same groups over and over again, or create new ones.
+ 
+*More insight into student work than ever before.* See when students accept the assignment, and access their work from the moment they start. With version control, catch when they get stuck and help them rewind. 
+ 
+*You are in control.* Students can work individually or in groups, in public or in private. Set permissions for teaching assistants or graders. 
+ 
+*Scales for large courses with ease.* If you have a small course, Classroom will make your life easier and save you time. f you have hundreds of students, we have your covered: as many repositories as you need, and webhooks to integrate automated testing tools. 
+ 
+*Works with your Learning Management System (LMS).* Students submit a link to their assignment repository to your learning management system. Give feedback through comments in GitHub, but keep grades in your LMS.
+ 
+*You’re not alone.* The experts on the GitHub Education team are here to answer any of your questions, and we’ve got docs, best practices, and a strong community of educators to help you migrate to Classroom.
+ 
+*Are you super-advanced?* Do you want to build your own tools? We 💖 contributions. 
+
+## Design principles
+
+*Classroom is a teacher-facing tool to enable the educational use of GitHub.* Every student needs feedback and support as they learn to code, and using GitHub you can give them the right advice, in the right place, at the right time. This tool makes it easier for you to use the workflow you love, but with some organization, automation and ease with your students. 
+ 
+*Students use GitHub. They don’t use Classroom.* Experience with real-world tools gives students a leg-up once they move on from school. Invest time in the tools students can grow with, not another third-party tool with its own learning curve. 
+ 
+*Classroom is not an LMS (Learning Management System).* If you need to use an LMS like Canvas, Moodle or Blackboard, we hear you. Students can post their repositories to your LMS. We’re going to stick with what we’re good at, which is helping people collaborate on code. 
+ 
+*Classroom is open source.* Git and GitHub are versatile with many ways to meet your learning goals, and we want to model the open source process that makes our communities great. 
+
+We welcome contributions aligned with the roadmap below and through answering these questions: 
+ 
+* Does it enable the real-life experience of using GitHub as a developer? 
+* Does it replicate functionality of apps, hardware or LMS? 
+* How many support tickets and questions will the feature address? 
+ 
+### Who is Classroom for?
+Anyone teaching computer science in a high school, university or informal environment.
```

Comment:
In the UK they're rolling out CS education for children beginning in Grade 1, so changing this to just say "schools" or something could be more accurate.